js/app.js

```diff
@@ -219,28 +256,67 @@ function pathIntoTitleInput(paste) {
 }
 
 function storeTags() {
+  const currentDomain = document.getElementById("domain-tag").innerHTML;
   const url = linkInput.value;
   const title = titleInput.value;
   let newTags = document.getElementsByClassName("tagged");
   let existTags = document.getElementsByClassName("exist-tagged");
-  //let largerArray = Math.max(newTags.length, existTags.length);
   let tileNo = Object.keys(STATE.tiles).length + 1;
   let newTile = "tile" + tileNo;
+  let domainNo = Object.keys(STATE.domains).length + 1;
+  let newDomain = "domain" + domainNo;
```

Comment:
Maybe adding a function to get/mount the new domain name

Reply:
After checking the domain STATE you don't need to create a new domain name as you have it already.